index.d.ts

```diff
@@ -128,6 +133,14 @@ export const Either: TypeRep;
 export function Left<A, B>(value: A): Either<A, B>;
 export function Right<A, B>(value: B): Either<A, B>;
 
+//  Placeholder
+
+interface Placeholder {
+  '@@functional/placeholder': boolean
```

Comment:
If I change `boolean` to `true`, this code example does not type check:

```typescript
import {concat} from 'sanctuary';

const _ = {'@@functional/placeholder': true};

concat(_, 'def')('abc');
```

Do you know what's wrong, @ikatyang? I'd love to use `true` if possible. :)